tools/start-cluster-servers.sh

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env bash
+#
```

Comment:
`tools/*` are lifted from typedb-client-nodejs and -python. Used to start and stop TypeDB for tests in CI.